.env

```diff
@@ -153,8 +153,22 @@ MF_CERTS_SIGN_RSA_BITS=2048
 MF_CERTS_VAULT_HOST=
 MF_CERTS_VAULT_PKI_PATH=pki_int
 MF_CERTS_VAULT_ROLE=agent
-MF_CERTS_VAULT_TOKEN=s.nArgw6xn3uIOfA7nfKk8LFaW
+MF_CERTS_VAULT_TOKEN=
 
+### Vault
+MF_VAULT_HOST=vault
+MF_VAULT_PORT=8200
+MF_VAULT_UNSEAL_KEY_1=
+MF_VAULT_UNSEAL_KEY_2=
+MF_VAULT_UNSEAL_KEY_3=
+MF_VAULT_TOKEN=
+MF_VAULT_CA_NAME=mainflux5
```

Comment:
are these default values right?

Reply:
I am using the same defaults you were using in https://github.com/mteodor/vault/blob/master/set-pki.sh#L11

Do you want me to change them? And if so, what to?

Reply:
ok, this like my 5th iteration, if it is not a problem just put `mainflux`